pkg/virt-launcher/virtwrap/converter/downwardmetrics.go

```diff
@@ -0,0 +1,38 @@
+package converter
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/downwardmetrics"
+
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+func convertDownwardMetricsChannel(downwardMetrics *v1.DownwardMetrics) []api.Channel {
```

Comment:
Can you please just return the `api.Channel`? I don't see the point in returning a slice with a single entry.

Reply:
Done.